docs/repositories-activity.html

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
re: filename
we should agree on a scheme going forward (`repositories` vs `repos`)

Reply:
agreed - I'm working on the new organizations (vs orgs) one now. ;-)

Reply:
@toddocon: You might want to have a look at [https://github.com/Autodesk/hubble/pull/27#discussion_r147785133](our recent discussion on this topic), where we agreed to be consistent with the GitHub API naming scheme.